src/app/students/GroupMembers.js

```diff
@@ -0,0 +1,87 @@
+import {withRouter} from "react-router";
+import {GroupIndexBanner} from "./GroupIndexBanner";
+import {ItemListPage} from "../commons/ItemListPage/ItemListPage";
+import {CreateButton} from "../commons/buttons/CreateButton";
+import FakeLink from "../commons/FakeLink";
+import React, {useEffect, useState} from "react";
+import {ThreeDotsButton} from "../commons/buttons/ThreeDotsButton";
+import {studentService} from "../../services/services";
+import {Spinner} from "../commons/Spinner";
+import {RemoveConfirmationModal} from "../commons/modals/RemoveConfirmationModal";
+
+
+const GroupMembers = withRouter(({history, match}) => {
+    const currentPathName = history.location.pathname;
+    const [group, setGroup] = useState(undefined);
+    const [students, setStudents] = useState(undefined);
+    const [member, setMember] = useState(undefined);
+    const [showRemoveMemberModal, setShowRemoveMemberModal] = useState(undefined);
+
+    const actionItemsButton = (member) => new ThreeDotsButton({
+        dropDownItems: [{
+            name: "Remove",
+            dangerous: true,
+            onClick: () => {
+                setMember(member.student);
+                setShowRemoveMemberModal(true);
+            }
+        }]
+    })
+
+    useEffect(() => {
+        if (!group) {
+            studentService.getGroupById(match.params.groupId)
+                .then(group => setGroup(group));
+        }
+
+        if (!students && group) {
+            studentService.getStudentsByGroupId(group.id)
+                .then(students => setStudents(students));
+        }
+    });
+
+    return (
+        <div>
+            {group === undefined ? <Spinner/> : <GroupIndexBanner currentPathName={currentPathName}
+                                                                  groupName={group.name}
+                                                                  groupId={group.id}/>}
```

Comment:
```js
 {group ? <GroupIndexBanner currentPathName={currentPathName} 
                                  groupName={group.name} groupId={group.id}/> : <Spinner/> }
```

( `GroupIndexBanner` should be renamed )

Reply:
@Chaoyuuu Unsolved

Reply:
![image](https://user-images.githubusercontent.com/41495987/114711138-2e5a7200-9d61-11eb-94c0-d15550e3596c.png)

I have renamed `GroupIndexBanner` to `GroupNavigationBar` in this pr.

Reply:
no